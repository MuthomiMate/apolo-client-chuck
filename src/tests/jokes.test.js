import React from "react";
import { shallow, configure } from "enzyme";
import Adapter from 'enzyme-adapter-react-16';
import Jokes from "../components/jokes";

configure({adapter: new Adapter()});
describe("<Jokes />", () => {
  test("should render without crashing", () => {
    const wrapper = shallow(<Jokes />);
    expect(wrapper).toMatchSnapshot();
  });
});