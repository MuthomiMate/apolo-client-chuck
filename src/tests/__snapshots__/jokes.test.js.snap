// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Jokes /> should render without crashing 1`] = `ShallowWrapper {}`;
